.appveyor.yml

```diff
@@ -27,28 +26,34 @@ install:
 - cmd: vcpkg upgrade --no-dry-run
```

Comment:
It would be better to replace the install+upgrade with
```
>vcpkg remove --recurse --outdated
>vcpkg install %ALL_PACKAGES%
```
Otherwise, you'll be rebuilding packages that you no longer use (like boost interprocess).

You should also run `./bootstrap-vcpkg.bat` after doing a `git pull`, because there may be important tool changes.

Reply:
Awesome

My original thought is that we could get rid of `boost-interprocess` by clear the cache, but if we use the command you provide, we don't have to clear the cache anymore. Just have to wait for next `boost-interprocess` update and it would be removed automatically.